gstd/src/common/errors.rs

```diff
@@ -40,6 +43,22 @@ impl fmt::Display for ContractError {
             ContractError::Decode(e) => write!(f, "Decoding codec bytes error: {}", e),
             ContractError::ExitCode(e) => write!(f, "Reply returned exit code {}", e),
             ContractError::Internal(e) => write!(f, "Internal error: {:?}", e),
+            // TODO: print error when it will contain actual information
+            ContractError::Send(_) => write!(f, "Send error"),
+            // TODO: print error when it will contain actual information
+            ContractError::Reply(_) => write!(f, "Reply error"),
         }
     }
 }
+
+impl From<SendError> for ContractError {
```

Comment:
How do you feel adding something like that to reduce amount of map and map_err with repeating Into::into?

```rust
impl<T, R, V> From<Result<T, R>> for Result<V>
where T: Into<V>, R: Into<ContractError>
```

Reply:
It violates orphan rules

Reply:
Yeah, but you can provide custom trait for it